src/cloudsc_loki/dwarf_cloudsc.F90

```diff
@@ -18,6 +18,7 @@ PROGRAM DWARF_CLOUDSC
 USE CLOUDSC_DRIVER_LOKI_MOD, ONLY: CLOUDSC_DRIVER
 #endif
 USE EC_PMON_MOD, ONLY: EC_PMON
+USE OMP_LIB
```

Comment:
Also, guard with `#ifdef _OPENMP`, and in l.51 below

Reply:
resolved in next commit